.github/workflows/ci.yaml

```diff
@@ -70,10 +73,129 @@ jobs:
 
       - name: Build/Tag/Push Image
         uses: docker/build-push-action@v2
+        id: docker_push
         with:
           context: .
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           file: ./dockerfiles/components/chainhook-node.dockerfile
           # Only push if (there's a new release on main branch, or if building a non-main branch) and (Only run on non-PR events or only PRs that aren't from forks)
           push: ${{ (github.ref != 'refs/heads/master' || steps.semantic.outputs.new_release_version != '') && (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) }}
+
+  deploy-dev:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
I like what you did with the matrix! However on the approval screen, it doesn't seem like it's going to let us differentiate between a testnet dev deployment and a mainnet dev deployment. So I think they only way to get around this is to create a total of 6 Github Environments, aptly named something like Development - Mainnet, Development - Testnet, etc. You might still be able to leverage the matrix usage here as well.

Reply:
Despite there being two deployments prepped, there's only one approval box. Ideally devs would have control to deploy to both networks for an environment, or just one.
<img width="670" alt="Screenshot 2023-09-29 at 8 39 30 PM" src="https://github.com/hirosystems/chainhook/assets/2747302/132222df-5a76-4b70-8e2d-7c7dc2323555">

Reply:
created 6 separate deployment environments as suggested and using matrix function